### DISCUSSION TITLE
makeAutoObservable but avoid members by type

### DISCUSSION THREAD

**Discussion by alisman** [mobxjs/mobx#3279] [Q&A]
I want to call makeAutoObservable but create an exception for fields based on their type.  Specifically, i want to avoid getters.  Is there a way to do this out of box?

**Comment by kubk** [comment]
What is the purpose of it?

**Comment by urugator** [comment]
No, `makeAutoObservable` is to provide convenience for simple/common cases, for anything else you have to be explicit.
